src/lib/dlt_user.c

```diff
@@ -2367,6 +2367,30 @@ DltReturnValue dlt_user_log_write_sized_utf8_string_attr(DltContextData *log, co
     return dlt_user_log_write_sized_string_utils_attr(log, text, length, UTF8_STRING, name, true);
 }
 
+DltReturnValue dlt_user_log_write_constant_utf8_string(DltContextData *log, const char *text)
+{
+    /* Send parameter only in verbose mode */
+    return is_verbose_mode(dlt_user.verbose_mode, log) ? dlt_user_log_write_utf8_string(log, text) : DLT_RETURN_OK;
```

Comment:
Better to return something like DLT_RETURN_ERROR or something in case of non-verbose mode so that application know that it was discarded

Reply:
I could certainly do that, but the pre-existing *_constant_* functions all return DLT_RETURN_OK in this situation. Changing those would be an incompatible change, and using DLT_RETURN_ERROR only for these new ones would be inconsistent with those others.

I wouldn't actually consider the discarding of these static strings to be an error - it is clearly documented that such static strings are not sent in non-verbose mode.

Reply:
Agreed. Then let's leave it as is.